### ISSUE TITLE
Line continuations can only work with spaces after them

### ISSUE THREAD

**Issue by ISSOtm** [rednex/rgbds#410]
```
issotm@link-panda tmp.kWMeWsvG5v% printf "PRINTT \\ \t\n\"Hello\"" | rgbasm -
error: Expected a new line after the continuation character.
```
Shouldn't `isspace(3)` be used instead?

**Comment by dbrotz** [comment]
I would avoid using `isspace()` because its behavior depends on the locale. RGBDS should have its own function to determine what characters are whitespace.

**Comment by ISSOtm** [comment]
Fixed in 6fbb25c

**Comment by ISSOtm** [comment]
Fixed in 6fbb25c
